src/components/photo_input.tsx

```diff
@@ -125,6 +125,7 @@ const PhotoInput: React.FC<PhotoInputProps> = ({
                     </Button>
                     <Button
                         variant="danger"
+                        data-testid="permanently-delete-button"
```

Comment:
Instead of adding the `data-testid` prop, could you find this element with a CSS selector, e.g., `button.btn-danger`?